pkg/app/piped/executor/cloudrun/cloudrun.go

```diff
@@ -158,6 +158,9 @@ func addBuiltinLabels(sm provider.ServiceManifest, hash, pipedID, appID, revisio
 	// Set builtinLabels for Service.
 	sm.AddLabels(labels)
 
+	if revisionName == "" {
+		return true
+	}
```

Comment:
I don't understand why do we need this?
If the `revisionName` is empty, the `sm.AddRevisionLabels(labels)` is not called, is that ok?

Reply:
If the revision have already existed, newRevision will be empty and this mean that needn't to set label again.
However, we can fix like this.

```
if newRevision != "" {
  if !addBuiltinLabels(sm, commit, e.PipedConfig.PipedID, e.Deployment.ApplicationId, newRevision, e.LogPersister) {
           return model.StageStatus_STAGE_FAILURE
  }
}
```
https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/executor/cloudrun/deploy.go#L189-L192

Reply:
I got it. So let's check it's empty at the caller side as you commented above.

Reply:
Allright, I addressed https://github.com/pipe-cd/pipecd/pull/3438/commits/c1550eb73f5d2e8c0fd45440f17a5cce99ea1b81